SecretParser.cs

```diff
@@ -4557,25 +4559,34 @@ public static void SetFrench()
             deAttacks.Add("Inversion de tendance");
             deAttacks.Add("Jet de pierre");
             deAttacks.Add("L'art de la guerre");
+			deAttacks.Add("Malédiction persistante");
+			deAttacks.Add("Maléfice sanglant");
             deAttacks.Add("Manifestation de feu");
             deAttacks.Add("Manifestation de foudre");
             deAttacks.Add("Manifestation de glace");
             deAttacks.Add("Marteau de Thor");
+			deAttacks.Add("Moisson de sang");
             deAttacks.Add("Onde de choc");
+			deAttacks.Add("Œil de la tempête");
             deAttacks.Add("Poussée de sang");
             deAttacks.Add("Rafale de trois balles");
             deAttacks.Add("Ralentissement de l'ennemi");
             deAttacks.Add("Règlement de comptes");
+			deAttacks.Add("Révélation d'essence");
             deAttacks.Add("Rideau de rubis");
             deAttacks.Add("Roue de poignards");
             deAttacks.Add("Sang de guerre");
+			deAttacks.Add("Symbole de terreur");
             deAttacks.Add("Taille de bambou");
             deAttacks.Add("Taille de brindille");
             deAttacks.Add("Tête de pont");
             deAttacks.Add("Tir de couverture");
             deAttacks.Add("Tir de précision");
+			deAttacks.Add("Tourment");

```

Comment:
I don't believe this is required.  This section should only have skills containing the word "de"